WordPress/Sniffs/XSS/EscapeOutputSniff.php

```diff
@@ -226,7 +226,7 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 			}
 
 			// Handle magic constants for debug functions.
-			if ( in_array( $tokens[ $i ]['code'], array( T_METHOD_C, T_FUNC_C, T_FILE, T_CLASS_C ), true ) ) {
+			if ( in_array( $tokens[ $i ]['code'], array( T_CLASS_C, T_FILE, T_FUNC_C, T_LINE, T_METHOD_C, T_NS_C, T_TRAIT_C ), true ) ) {
```

Comment:
When I made the change I checked that:
a) there wasn't already a (static) property in the PHPCS Tokens class we could use for this (there isn't)
b) whether any of these tokens are used anywhere else in WPCS (they are not).

I thought about making it a property, but am a bit wary what with the fact that the actual values these constants represent are integers which change between PHP versions. It shouldn't be an issue, but still.

Thinking now - I might as well make it a property with the string name as keys and check against the `$tokens[ $i ]['type']` value instead of `code`. That should work and be stable.